### PR TITLE
Handle time offset in EPG programme start and end times

### PR DESCRIPTION
**Pull request by ChadiEM**
To solve issue #85 .

### CODE REVIEW

**Comment by Jalle19** [comment]
Can you bump the version to 2.4.4 and update the changelog too? See https://github.com/kodi-pvr/pvr.iptvsimple/commit/6ea17c50765ad4ae0f661b5c5847deea3589200e for an example.

**Comment by Jalle19** [comment]
Thanks!

File: src/PVRIptvData.cpp [hunk 1]
```diff
@@ -749,17 +749,29 @@ int PVRIptvData::ParseDateTime(std::string& strDate, bool iDateFormat)
...
     sscanf(strDate.c_str(), "%02d.%02d.%04d%02d:%02d:%02d", &timeinfo.tm_mday, &timeinfo.tm_mon, &timeinfo.tm_year, &timeinfo.tm_hour, &timeinfo.tm_min, &timeinfo.tm_sec);
 
   timeinfo.tm_mon  -= 1;
   timeinfo.tm_year -= 1900;
   timeinfo.tm_isdst = -1;
 
-  return mktime(&timeinfo);
+  std::time_t current_time;
+  std::time(&current_time);
+  long offset = std::localtime(&current_time)->tm_gmtoff;
```
Comment (afedchin) [inline src/PVRIptvData.cpp]: `tm_gmtoff` is not available on WIN32. please fix this. or I will revert this.

**Comment by Jalle19** [comment]
Typical, I just trusted Travis.

**Comment by ChadiEM** [comment]
So did I. Thanks for letting me know @afedchin. Unfortunately, I don't have a Windows machine to develop this change on, so please feel free to amend the change or revert it completely.

**Comment by afedchin** [comment]
fixed already in #90

**Pull request merged**